File: demos/jans-chip/app/src/main/java/io/jans/chip/modal/TokenResponse.java
package io.jans.chip.modal;

import com.google.gson.annotations.SerializedName;

public class TokenResponse {
    @SerializedName("access_token")
    private String accessToken;
    @SerializedName("id_token")
    private String idToken;
    @SerializedName("token_type")
    private String tokenType;

    public String getTokenType() {
        return tokenType;
    }

    public void setTokenType(String tokenType) {
        this.tokenType = tokenType;
    }

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public String getIdToken() {
        return idToken;
    }

    public void setIdToken(String idToken) {
        this.idToken = idToken;
    }
}

File: demos/jans-chip/android/app/src/main/java/io/jans/chip/model/TokenResponse.kt
package io.jans.chip.model

import androidx.room.Ignore
import com.google.gson.annotations.SerializedName

data class TokenResponse (

    @SerializedName("access_token")
    var accessToken: String? = null,
    @SerializedName("id_token")
    var idToken: String? = null,
    @SerializedName("token_type")
    var tokenType: String? = null
){
    @Ignore
    var isSuccessful: Boolean? = true

    @Ignore
    var errorMessage: String? = null
}